### ISSUE TITLE
Keybindings

### ISSUE THREAD

**Issue by einveru** [babluboy/bookworm#231]
System:
Arch Linux / i3

Is there currently a way to alter the keybindings? I've gotten accustomed to 'vim' keybindings and would prefer to move up/down pages using j/k instead of up/down.

**Comment by babluboy** [comment]
@einveru Many thanks for your interest in Bookworm and raising this feature request.

At the moment there is no option for the user to alter the keybindings. But I can add this feature using the Elementary Key Binding widget. Will use this issue to track this feature.

**Comment by einveru** [comment]
Are you planning on developing 'bookworm' for use outside of Elementary OS? For instance, will I be able to configure the bindings using the Elementary Key Binding widget using other distros and DE's? Great program btw. Going to be my go-to for ebooks.

**Comment by babluboy** [comment]
Yes...my longer term goal is to make Bookworm compatible with as many distros as possible. 
The Elementary keybinding is a granite widget and should work on any distro ...

https://cdn-images-1.medium.com/max/1600/1*BvnFc7KbHh55SFRRrGL0Ig.png
